integration_tests/packages.yml

```diff
@@ -2,8 +2,10 @@ packages:
   - package: fivetran/ad_reporting
     version: [">=1.0.0", "<2.0.0"]
 
-  - package: fivetran/shopify_holistic_reporting
-    version: [">=0.1.0", "<1.0.0"]
+  # - package: fivetran/shopify_holistic_reporting
+  #   version: [">=0.1.0", "<1.0.0"]
+  - git: https://github.com/fivetran/dbt_shopify_holistic_reporting.git
+    revision: feature/bump-packages
```

Comment:
Would it be possible to swap this before release?

Reply:
Only if `dbt_shopify_holistic_reporting` is updated with the newest shopify packages.